content/collective_intro.tex

```diff
@@ -16,29 +60,29 @@
 satisfy the requirement that its last member corresponds to a valid \ac{PE}
 number, that is
 $0 \le PE\_start + (PE\_size - 1) * 2^{logPE\_stride} < npes$.
-All \acp{PE} participating in the collective routine must provide the same
+All \acp{PE} participating in the \newtext{active set-based} collective routine must provide the same
```

Comment:
The team-based, active set-based is a bit hard to parse. Changing from "active set-based" to "active-set-based" may be more readable for this phrasing. Or even "non-teams collectives" could work. Also, some structural changes may help, e.g. putting discussion of active sets in a separate subsection that can be easily deprecated out.

Reply:
> The team-based, active set-based is a bit hard to parse.

👍 on "active-set-based". I'm not crazy about "non-teams collectives" because routines like `shmem_barrier_all()` would fall under that, and I'd like to handle those separately (e.g., by defining them in terms of team-based collectives).

The main reason I didn't make an explicit subsection for discussion of active sets was that no other API section introduction has subsections. It wouldn't be a terrible thing, but I was trying stay consistent with the existing document structure. With the existing deprecation markings, I think it's pretty clear about which text is deprecated.

Reply:
Agreed about the point on "non-teams" being misleading due to the "*_all" collectives being team-based under the covers. Also agreed that deprecation markings should be good enough. Plus this avoids actually adding new structure and content into something that is going to be removed. I think it would help to have a sentence or two at the top of the intro to mention that there are two types of collectives. I will add an in-line comment there....